enclave-tls/src/crypto_wrappers/internal/etls_crypto_wrapper_load_single.c

```diff
@@ -29,20 +32,20 @@ enclave_tls_err_t etls_crypto_wrapper_load_single(const char *fname)
 		return -ENCLAVE_TLS_ERR_INVALID;
 	}
 
-	char realpath[strlen(CRYPTO_WRAPPERS_DIR) + strlen(fname) + 1];
-	sprintf(realpath, "%s%s", CRYPTO_WRAPPERS_DIR, fname);
-
-	void *handle = dlopen(realpath, RTLD_LAZY);
-	if (!handle) {
-		ETLS_ERR("failed on dlopen(): %s\n", dlerror());
-		return -ENCLAVE_TLS_ERR_DLOPEN;
-	}
+	size_t realpath_len = strlen(CRYPTO_WRAPPERS_DIR) + strlen(fname) + 1;
+	char realpath[realpath_len];
+	snprintf(realpath, realpath_len, "%s%s",CRYPTO_WRAPPERS_DIR, fname);
```

Comment:
you can use `snprintf(realpath, sizeof(realpath), ...)` to avoid using realpath_len.